rockspec/apisix-master-0.rockspec

```diff
@@ -52,6 +52,7 @@ dependencies = {
     "lua-resty-kafka = 0.07",
     "lua-resty-logger-socket = 2.0-0",
     "skywalking-nginx-lua-plugin = 1.0-0",
+    "base64 = 1.5-2"
```

Comment:
we don't need this dependency, you can use ngx.encode_base64 and ngx.decode_base64

Reply:
same as https://github.com/apache/apisix/pull/2036/files#r487795007